### PR TITLE
Declarations

### PR DESCRIPTION
**Pull request by renatoi**
@src-code @thierryk

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/atomizer.js [hunk 1]
```diff
@@ -587,67 +395,45 @@ Atomizer.prototype.parseConfig = function (config/*:AtomizerConfig*/, options/*:
...
-                                        }
-                                    });
-                                });
-                                namedFound = true;
-                                return true;
-                            }
-                        });
+                    // first check if the named value matches any of the values
+                    // registered in arguments.
+                    if (rule.arguments && index <= rule.arguments.length && Object.keys(rule.arguments[index]).indexOf(matchVal.named) >= 0) {
```
Comment (src-code) [inline src/atomizer.js]: `Object.keys` support is limited to IE9+ client side, so if we want to support IE7-8, we'll want to use lodash here.

**Comment by src-code** [comment]
LGTM.

**Comment by 3den** [comment]
awesome :+1:

**Pull request merged**